agreement_sale/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Agreement - Sale',
+    'summary': 'Create an agreement when the sales order is confirmed',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Open Source Integrators, Odoo Community Association (OCA)',
+    'category': 'Agreement',
+    'website': 'https://github.com/OCA/contract',
+    'depends': [
+        'agreement_legal',
```

Comment:
`agreement` and `sale`, not `agreement_legal`

Reply:
But [this view](https://github.com/OCA/contract/pull/292/files#diff-13b9083d98e59c16fb7d525111063146R12) is created in agreement_legal module.

Reply:
The `agreement` module should define the notebook (and it could be empty) and the `agreement_sale` one should add a page to it.